virttest/libvirt_xml/vm_xml.py

```diff
@@ -2733,14 +2733,17 @@ class VMCPUTuneXML(base.LibvirtXMLBase):
         quota:                int
         emulator_period:      int
         emulator_quota:       int
+        global_period:        int
```

Comment:
the parameter in docstring doesn't match with ones in __slots__

Reply:
@chunfuwen updated.